README.md

```diff
@@ -19,10 +20,49 @@ via autowiring.
 
 ## Features
 
-- Invoke callable or create an object of a given class.
-- Resolve dependencies by parameter types using the given [PSR-11](http://www.php-fig.org/psr/psr-11/) container.
-- Pass concrete dependency instances by type.
```

Comment:
This fact is lost.

Reply:
That's part of "Resolves dependency objects by parameter" combined with "Accepts additional custom arguments from an array" below

Reply:
That doesn't reflect the original. It was about the ability to do something like this:

```php
function test(CacheInterface $cache, LoggerInterface $log) { ... }

$injector = new Injector($container);
echo $injector->invoke('test', [new MyLogger()]);
```

In this case it will find matching parameter (logger) by type while obtaining the rest from the container.